demos/demo-react-ts/src/components/App.tsx

```diff
@@ -46,24 +46,32 @@ export const formatTime = (totalSeconds: number) => {
 };
 
 function App() {
-  const { cti, phoneNumber, engagementId, callStatus, incomingContactName } =
-    useCti();
   const [screenIndex, setScreenIndex] = useState(0);
   const [direction, setDirection] = useState<Direction>("OUTBOUND");
   const [dialNumber, setDialNumber] = useState("+1");
   const [notes, setNotes] = useState("");
+  const [showAlert, setShowAlert] = useState(true);
+  const [fromNumber, setFromNumber] = useState("+1 617-948-3986");
+  const [incomingNumber, setIncomingNumber] = useState("+1");
+  const [availability, setAvailability] = useState<Availability>("UNAVAILABLE");
+
   const {
     callDuration,
     callDurationString,
     startTimer,
     stopTimer,
     resetCallDuration,
   } = useCallDurationTimer();
-  const [showAlert, setShowAlert] = useState(true);
-  const [fromNumber, setFromNumber] = useState("+1 617-948-3986");
-  const [incomingNumber, setIncomingNumber] = useState("+1");
 
-  const [availability, setAvailability] = useState<Availability>("UNAVAILABLE");
+  const initializeCallingStateForExistingCall = () => {
```

Comment:
nit: Usually, it is a good idea to keep all hook calls at the top of the function. So that, we dont accidentally introduce a condition in between. React will complain about it. So, i think we can move this function below useCti hook call

Reply:
I moved the useCti hook below as we need to call setState in the hook itself. And we have to define the `initializeCallingStateForExistingCall` function before calling it in useCti.